client/dashboard/ciab-sites/index.tsx

```diff
@@ -64,6 +66,8 @@ export default function CIABSites() {
 	const isRestoringAccount = !! currentSearchParams.restored;
 
 	const { user } = useAuth();
+	const { onboardingLinks } = useAppContext();
```

Comment:
I think we don't need this anymore as the CIAB won't use the shared `AddNewSiteModal`